src/kOS/Communication/RunCommand.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+using kOS.Safe.Compilation;
+
+namespace kOS.Communication
+{
+    public class RunCommand : InterProcCommand
+    {
+        public List<Opcode> Program;
+
+        public override void Execute(SharedObjects shared)
+        {
+            if (shared.Cpu != null)
+            {
+                shared.Cpu.RunProgram(Program);
+            }
+        }
+    }
+}
```

Comment:
I've noticed that there is an inter-cpu command for running a program but I don't see any documentation of such a feature.  I thought that we had discussed this option and weren't going to include that feature, that all of the communication would be done via messages and that enacting commands on other processors would have to be managed by a listening script.  Am I mis-remembering?